src/v1_4/Differ.php

```diff
@@ -232,7 +232,7 @@ public function diffToArray($from, $to, LongestCommonSubsequence $lcs = null)
 
         if ($this->detectUnmatchedLineEndings($fromMatches, $toMatches)) {
             $diff[] = array(
-                '#Warning: Strings contain different line endings!',
```

Comment:
@keradus why was change needed for v3.0?

Reply:
i copy-pasted v1 from original repo as well, that was the change from there